### ISSUE TITLE
Colour contrast between text and background does not meet accessibility standards. 

### ISSUE THREAD

**Issue by brainlava** [BSiLabs/RateTheMeet#1]
Throughout the app, the contrast between the text colours and the background colours do not pass accessibility standards.  For example the colour of "rate" on the blue background of the loading screen. Or Current Surveys in the header.  

All text and background colour combinations should be vetted using the following site: https://webaim.org/resources/contrastchecker/

If the contrast ratio is not met it will need to be adjusted either darker or lighter.

![image](https://user-images.githubusercontent.com/13080349/58119854-faf12600-7bd1-11e9-9acb-dbb70b59d7f3.png)
![image](https://user-images.githubusercontent.com/13080349/58119887-0a706f00-7bd2-11e9-98be-ac3c418290aa.png)
![image](https://user-images.githubusercontent.com/13080349/58119869-02b0ca80-7bd2-11e9-8e97-6b8efef17742.png)
